assets/src/blocks/Articles/ArticlesFrontend.js

```diff
@@ -33,26 +31,26 @@ export const ArticlesFrontend = (props) => {
         <h2 className="page-section-header">{ article_heading }</h2>
       </header>
       { articles_description &&
-        <div className="page-section-description" dangerouslySetInnerHTML={{ __html: articles_description }} />
+        <div className="page-section-description" dangerouslySetInnerHTML={{__html: articles_description}} />
       }
-      <ArticlesList posts={ posts } postType={ postType }/>
+      <ArticlesList posts={posts} postType={postType} />
       { hasMorePages &&
       <div className="row">
-        { read_more_link ?
-          <div className="col-md-12 col-lg-5 col-xl-5">
+        { read_more_link
+          ? <div className="col-md-12 col-lg-5 col-xl-5">
             <a
               className="btn btn-secondary btn-block article-load-more"
-              href={ read_more_link }
-              target={ button_link_new_tab ? '_blank' : '' }
+              href={read_more_link}
+              target={button_link_new_tab ? '_blank' : ''} rel="noreferrer"
```

Comment:
Shouldn't this `rel="noreferrer"` be added only when target is blank?

Reply:
It should, but we are conditionally adding target blank but ESLint does not know that so it throws an error because it sees the target attribute.